### DISCUSSION TITLE
Multilanguage support

### DISCUSSION THREAD

**Discussion by Curiosit** [linkarkitektur/SpeckleLCA#9] [Ideas]
I have modified the LCAByg constructions library file to have multiple language names. I think we should keep at least english and danish (as the elements in LCAByg library are in danish). Would be nice to also add more possible languages later.
I have modified the view of material mapper to show double names (english and danish)

**Comment by pfmephisto** [comment]
I guess there are some libraries that help manage translations and the loading of languages.
Guess I can take a look at this at some point at least how to structure it technically.

**Reply by pfmephisto** [reply]
Looks linke this might be worth looking at.
https://vue-i18n.intlify.dev/

**Reply by Curiosit** [reply]
This could be a nice solution for translating the webpage - but this is less important I think as this can be kept mainly in english? I thought more about structuring data so that we can 'generate' JSON LCAByg files in different languages - danish, english, swedish, polish etc. - because that is the file we often give to some other person, or display results from - and then localization is much more important I think
